react/components/ProductDescription/index.js

```diff
@@ -11,21 +11,29 @@ import styles from './styles.css'
  * Product Description Component.
  * Render the description of a product
  */
-const ProductDescription = ({ description }) => {
+const ProductDescription = ({ description, showShowMoreButton }) => {
```

Comment:
I believe the name `showShowMoreButton` could be better, maybe `showMoreButtonVisible`?

Reply:
Bruzzi, para programadores sua sugestão é ótima, porém essa é uma variável setada no arquivo de configuração por um leigo, então eu acho que para esse público `showShowMoreButton` é bem mais claro no que ela faz.